backend/routes.js

```diff
@@ -179,12 +179,25 @@ route.GET('/assets/{path*}', {
         return h.continue
       }
     }
+  },
+  files: {
+    relativeTo: path.resolve('dist/assets')
   }
-}, {
-  directory: {
-    path: path.resolve('./dist/assets'),
-    redirectToSlash: true
+}, function (request, h) {
+  const { subpath } = request.params
+  const basename = path.basename(subpath)
+  const extension = path.extname(subpath)
+  console.debug(`GET /assets/${subpath}`)
+  // In the build config we told our bundler to use the `-cached-[hash]` template to name immutable assets.
+  // This is useful because `dist/assets/` currently includes a few files without hash in their name.
+  if (basename.endsWith(`-cached${extension}`)) {
+    return h.file(subpath, { etagMethod: false })
+      .etag(basename.slice(0, -extension.length))
```

Comment:
Oh, while you're updating that, this isn't a bug, but I think it would simplify and slightly speed up things here to just pass in the `basename` directly for the `.etag` here

Reply:
Yea that would be simpler, however this also results in more bytes to be sent. Since the full extension if difficult to extract (it would require a regex), I now think that stripping the `-chunk` part instead is a good compromise.